docs/pages/web/zindex_classes.js

```diff
@@ -230,7 +230,7 @@ const canvas = (
             cardSize="lg"
             type="do"
             description={`
-Use Layer instead of z-index when a component needs to visually break out of its parent container, for example, in the case of [modals](/web/modal), [sheets](/web/sheet) and [tooltips](/web/tooltip). See [z-Index in foundational components](#zIndex-in-foundational-components) and [ZIndex in Layer](#zIndex-in-Layer) to learn more.
+Use Layer instead of z-index when a component needs to visually break out of its parent container, for example, in the case of [modals](/web/modal), [overlay panels](/web/overlaypanel) and [tooltips](/web/tooltip). See [z-Index in foundational components](#zIndex-in-foundational-components) and [ZIndex in Layer](#zIndex-in-Layer) to learn more.
```

Comment:
since we're referring to the specific components (hence the links), these should all be PascalCase

Reply:
DONE